ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiConfigManagementService.scala

```diff
@@ -128,7 +129,7 @@ private[apiserver] final class ApiConfigManagementService private (
           entry <- synchronousResponse.submitAndWait(
             submissionId,
             (params.maximumRecordTime, newConfig),
-          )(executionContext, materializer)
+          )(NoOpTelemetryContext, executionContext, materializer)
```

Comment:
When are you planning on removing these `NoOpTelemetryContext` values?

Reply:
Next two PRs tomorrow :)

Reply:
:hedgehog: